apps/docs/src/app/core/component-docs/file-uploader/examples/file-uploader-max-example/file-uploader-max-example.component.ts

```diff
@@ -9,14 +9,12 @@ export class FileUploaderMaxExampleComponent {
     files: File[];
     invalidFiles: File[];
 
-    handleFileSection(): void {
-
-        alert('Files selected successfully!!!');
-
+    handleFileSection(files: File[]): void {
+        alert(files.length + ' Files selected successfully!!!');
```

Comment:
I don't think this should be hardcoded. What if there's i18n and you need to display this in another language?